.vscode/settings.json

```diff
@@ -1,3 +0,0 @@
-{
-    "terminal.integrated.fontFamily": "Hack Nerd Font"
-}
```

Comment:
I think this is not related to the actual codebase, this might be a case where we want to have this file in .gitignore

Reply:
Working on it.